src/screens/OpeningScreen/components/GameDisplay/GameScreen/PlayerCurrentInteraction/PcInteraction/Pokedex/PokedexSingleRow/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { black } from 'assets/AnimationOpening/stylesColorsCode';
+import React, { memo } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { WINDOW_WIDTH } from 'src/screens/OpeningScreen/components/GameDisplay';
+
+interface PokemonData {
+  number: string;
+  name: string;
+}
+
+const singleRowContainerText = {
+  fontFamily: 'PokemonClassic',
+  fontSize: 10,
+  color: black,
+  textAlign: 'left' as const,
+};
+
+const PokedexSingleRowComponent = ({ number, name }: PokemonData) => {
+  return (
+    <View style={styles.singleRowContainer}>
+      <Text style={styles.pokemonNumber}>No {number}</Text>
```

Comment:
```suggestion
      <Text style={[styles.singleRowContainer, styles.pokemonNumber]}>No {number}</Text>
```
utiliser la syntaxe mise a notre disposition par react plutot que de spread une variable dans ton style, fonctionnellement c'est identique mais c'est plus lisible de passer directement par la balise Text et son attribut style

Reply:
ok fait